platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -264,6 +264,34 @@ sep1:
     #
     value: |
 
+sep6:
+  ## @param: enabled
+  ## @type: bool
+  ## Whether to enable SEP-6 for this instance of AP server.
+  #
+  enabled: false
+  ## @param: transactions
+  ## Configures the GET transactions endpoint
+  #
+  transactions:
+    ## @param: enabled
+    ## Whether to enable the GET transactions endpoint
+    #
+    enabled: true
+  ## @param: transaction
+  ## Configures the GET transaction endpoint
+  #
+  transaction:
+    ## @param: enabled
+    ## Whether to enable the GET transaction endpoint
+    #
+    enabled: true
+  # Configures the features flag returned by the SEP-6 /info endpoint
+  # For details, please refer to https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#response-4
+  #
+  features:
```

Comment:
We are not supporting the `account_creation` and `claimable_balances` in the first version. We do not need to configure these flags until we decide to support them and they may need extra fields.

Reply:
If I'm not mistaken, we don't support these in SEP-24 either but we still allow partners to configure this. I can remove this though.

Reply:
I think we should keep them because they are used in softserve's work with Fireblocks. If fireblocks is configured then account creation is supported. Alternatively we can remove them and ask softserve to add them back as a part of their changes.